src/IIIFPresentation/Repository/Paths/IPathGenerator.cs

```diff
@@ -99,4 +99,6 @@ Uri GenerateFlatCollectionViewLast(Collection collection, int lastPage, int page
     /// Generate the hierarchical id for specified customer and path slugs 
     /// </summary>
     string GenerateHierarchicalFromFullPath(int customerId, string? fullPath);
+
+    string? GetModifiedImageRequest(string? existing, int customerId, int spaceId, int width, int height);
```

Comment:
Missing XML comment, everything else in class has brief one explaining what's being returned